src/app/DebugImages.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import { ScrollView } from 'react-native-gesture-handler'
+import LinearGradient from 'react-native-linear-gradient'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import * as Images from 'src/images/Images'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+// @ts-expect-error property "context" does not exist in the NodeRequire type,
+// but it is available in the metro runtime
+const icons = require.context('../icons', true, /\.tsx$/)
+// @ts-expect-error
+const images = require.context('../images', true, /\.tsx$/)
+
+const ICON_SIZE = 32
+const IMAGE_SIZE = 96
+
+const IconItem = ({ Component, fileName }: { Component: React.ElementType; fileName: string }) => {
+  return (
+    <View style={styles.itemContainer}>
+      {/* Not all icons have the same props, we do our best here to set consistent size and color */}
+      <Component height={ICON_SIZE} width={ICON_SIZE} size={ICON_SIZE} color={Colors.black} />
+      <Text>{fileName.split('.tsx')[0].slice(2)}</Text>
+    </View>
+  )
+}
+
+function DebugImages() {
+  return (
+    <LinearGradient
+      colors={[Colors.white, Colors.gray3]}
+      start={{ x: 0, y: 0 }}
+      end={{ x: 0, y: 1 }}
+    >
+      <ScrollView>
```

Comment:
nit: I thought ScrollView would struggle rendering all these views at once.

Reply:
yeah maybe there might be performance issues if running on a device, but i figure this screen will be most useful when building new features and running on our laptops. i'll keep an eye on it in the nightly and happy to update it to use flatlist or something if there are issues, but i wanted to keep it fast and simple for now